### ISSUE TITLE
Loss curves for training and test/validation set

### ISSUE THREAD

**Issue by YKritet** [AlexeyAB/darknet#3446]
I would like to have access to the curves describing the evolution of loss (and the accuracy if possible) for both the training set as well as for the validation/test set. The "chart.png" file shows only the loss without specifying whether it's the validation or the training set's. 

I can't tell whether the model have learned how to detect an object or it is only overfitting my data otherwise. Ideally, I want to draw the loss and accuracy curves for both the training and the validation set which makes for 4 curves. Thanks in advance.
